docs/src/views/FAQ.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react'
+import DocPage from '../components/DocPage/DocPage'
+import GuidesNavigationFooter from '../components/GuidesNavigationFooter'
+import CodeSnippet from '../components/CodeSnippet'
+
+import { Header } from '@stardust-ui/react'
+
+export default () => (
+  <DocPage title="FAQ">
```

Comment:
would be great to add content at the beginning with links to a specific heading "How can ...".
As a FAQ will grow, it is an efficient way for users to go through the questions and jump to the one they're interested in. An example from Accessibility page:

![image](https://user-images.githubusercontent.com/8460706/56650515-7f11c580-6687-11e9-9a8e-b35860b65028.png)

Reply:
💯 for this

Reply:
let's do later when we have enough content to get past one screen ;)